pages/idea/[id].tsx

```diff
@@ -47,7 +48,6 @@ const ProfileLink = ({ id }: { id: string }) => {
   const { data: creatorEns } = useEnsName({
     address: id as `0x${string}`,
     cacheTime: 6_000,
-    suspense: true,
```

Comment:
After commenting out a bunch of code it seems like this was the cause of the error. It would only happen on the first page load or before the ENS name has been cached.